pydantic/typing.py

```diff
@@ -121,12 +121,9 @@ def get_args(tp: Type[Any]) -> Tuple[Any, ...]:
             get_args(Union[int, Tuple[T, int]][str]) == (int, Tuple[str, int])
             get_args(Callable[[], T][int]) == ([], int)
         """
-        try:
-            args = typing_get_args(tp)
-        except IndexError:
-            args = ()
+        args = typing_get_args(tp)
```

Comment:
@PrettyWood I'm not sure about this, I couldn't find a case where `IndexError` is raised, hence removing it.

Do you know what you did to cause the error?

Reply:
You're absolutely right ! I did this before splitting `python 3.8` and `python 3.9`.
Now that it's under `sys.version_info >= (3, 9)` it can be removed !